diff.py

```diff
@@ -1730,6 +1805,76 @@ class ArchSettings:
     "bgt-",
 }
 
+I686_BRANCH_INSTRUCTIONS = {
+    "call",
```

Comment:
mips doesn't include its function call instruction here fwiw. I think currently we might crash if a "branch instruction" has a relocation, because we try to parsed the relocated symbol as a base 16 address? which would be possible to fix ofc, just something you might run into